test/fixtures/app/nested-msgs/locale/fr/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,6 @@
+msgid ""
```

Comment:
A personal preference: I like it when the tests' sample app models real word example.  `nested-msgs` does not mean much to me in terms of "realworldness".  Recommending `installed_deps/foo`